src/cli.rs

```diff
@@ -161,6 +161,11 @@ pub struct RootOpts {
     #[cfg(feature = "allocation-tracing")]
     #[arg(long, env = "ALLOCATION_TRACING", default_value = "false")]
     pub allocation_tracing: bool,
+
+    /// Set allocation tracing reporting rate in milliseconds.
+    #[cfg(feature = "allocation-tracing")]
+    #[arg(long, env = "ALLOCATION_TRACING_RATE", default_value = "5000")]
```

Comment:
This should be `allocation_tracing_reporting_interval_ms` and `ALLOCATION_TRACING_REPORTING_INTERVAL_MS`, respectively.

The current name, by itself, doesn't mention the units involved _and_ sort of implies that it changes the rate of tracing itself, like a simple rate or something.